relocation.bs

```diff
@@ -1021,6 +1040,60 @@ is destructor is called and the exception is propagated.
 Note that the target object needs not to be destroyed as the delegating 
 constructor already took care of that.
 
+### Invocation
+
+The relocating constructor is invoked *as necessary* to relocate a prvalue
+from one storage location to another. Use of the `reloc` operator does not
+guarantee that the relocation constructor will be called, since it may be elided
+if the compiler can arrange that the source glvalue was constructed at the
+appropriate address.
```

Comment:
Also, the class may not declare any relocation constructor, and `reloc` will fallback on move+destroy or copy+destroy.